include/tallies/zernike_polynomial.hpp

```diff
@@ -0,0 +1,75 @@
+#ifndef ZERNIKE_POLYNOMIAL_H
+#define ZERNIKE_POLYNOMIAL_H
+
+#include <math.h>
```

Comment:
```suggestion
#include <cmath>
```
Always use the C++ style includes.